src/lib/serviceRoutinesV2/postEntities.cpp

```diff
@@ -78,10 +79,20 @@ std::string postEntities
     ciP->httpHeaderValue.push_back(location);
     ciP->httpStatusCode = SccCreated;
   }
+  else
```

Comment:
Maybe is safer to check the error condition instead of indiscrimined `else`. I mean:

```
else if ((ciP->httpStatusCode == SccUnprocessableEntity    )
```